pkgs/development/python-modules/cleo/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi,
+  pylev, pastel, clikit }:
+buildPythonPackage rec {
+  pname = "cleo";
+  version = "0.7.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "091nzpfp5incd2fzqych78rvyx4i3djr50cnizbjzr3dc7g00l3s";
+  };
+
+  propagatedBuildInputs = [
+    pylev
+    pastel
+    clikit
+  ];
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    homepage = https://github.com/sdispater/cleo;
+    description = "Cleo allows you to create beautiful and testable command-line interfaces";
```

```suggestion
    description = "Allows you to create beautiful and testable command-line interfaces";
```

We don't like to have the package name in the `description`.